### DISCUSSION TITLE
The UI Overflow on smaller screens, how to fix it?

### DISCUSSION THREAD

**Discussion by therohitdas** [saadeghi/daisyui#2888] [Q&A] [answered]
Hi, I am using default DaisyUI components like HERO, NAV etc.
The UI is overflowing and is not responsive on anything below 330 width.
Hero background is not taking full width on these smaller screens.

![Screenshot 2024-03-06 at 5 25 02 PM](https://github.com/saadeghi/daisyui/assets/43847374/918023cb-55bc-4aa6-9d29-e89b5ef2ea7e)

Can anyone help me point in the right direction to make the UI responsive on the small screen.

Stackblitz URL to preview and edit code:
https://stackblitz.com/edit/stackblitz-starters-a18ktt?file=index.html

**Comment by saadeghi** [comment, marked as answer]
This is not related to daisyUI.  
Remove those `text-nowrap` class names you used.  

Also, you don't need to design websites for `215px` width unless you want to use the website on a smart watch or something 😅
